lib/fastlane/plugin/mango/helper/emulator_commander.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'docker_commander'
+
+module Fastlane
+  module Helper
+    module EmulatorCommander
+      # Disables animation for faster and stable testing
+      def self.disable_animations(container_name:)
+        DockerCommander.docker_exec(command: 'adb shell settings put global window_animation_scale 0.0', container_name: container_name)
+        DockerCommander.docker_exec(command: 'adb shell settings put global transition_animation_scale 0.0', container_name: container_name)
+        DockerCommander.docker_exec(command: 'adb shell settings put global animator_duration_scale 0.0', container_name: container_name)
+      end
+
+      # Increases logcat storage
+      def self.increase_logcat_storage(container_name:)
+        DockerCommander.docker_exec(command: 'adb logcat -G 16m', container_name: container_name)
+      end
+
+      # Restarts adb on the separate port and checks if created emulator is connected
+      def self.check_emulator_connection(container_name:)
+        UI.success('Checking if emulator is connected to ADB.')
+
+        if emulator_is_healthy?(container_name: container_name)
+          UI.success('Emulator connected successfully')
+        else
+          raise "Something went wrong. Newly created device couldn't connect to the adb"
```

Comment:
Would it not be better if this would just return true or false?
It could then be up to the caller to determine if this is something unexpected or maybe wait and try again.